### DISCUSSION TITLE
Alert does not have the alert role

### DISCUSSION THREAD

**Discussion by apoco** [mantinedev/mantine#1263] [Feedback]
I'm not sure if this is a bug or if I'm misunderstanding something.

The [docs for the `Alert` component](https://mantine.dev/core/alert/) say in the accessibility section:

> Root element role set to `alert`

...but neither in the example HTML nor in my app's `Alert` does the `alert` role show up. I'm not sure if the comment is saying "for accessibility, set the role to `alert`, which I can do, or if this is a bug where the component is supposed to set that role automatically and it isn't for some reason. Ideally I'd want `<Alert/>` components to have the `alert` role by default. Thoughts?

**Comment by rtivital** [comment]
Thanks for reporting, the issue will be fixed with next patch.

**Reply by apoco** [reply]
I created a pull request to hopefully save you some time:

https://github.com/mantinedev/mantine/pull/1269
